lib/stats/icereport.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+/**
+ * @property {number} [availableSend] - bps (undefined in Firefox)
+ * @property {number} recv - bps
+ * @property {number} [rtt] - s (undefined in Firefox)
+ * @property {number} send - bps
+ */
+class IceReport {
```

Comment:
@markandrus Can't we re-use our Active Ice Candidate Pair stats that we already collect? Or are these stats different?

Reply:
IceReport is computing an additional piece of information beyond the Active ICE Candidate Pair, `bps`; however, we _should_ identify a way to share the stats we're collecting between Insights and Network Quality API. Ideally we'd just share an RTCStatsReport between both, but Firefox complicates this, since there's no way to tie RTCStats instances to particular RTCRtpSenders/RTCRtpReceivers/MediaStreamTracks unless you call `getStats` on those objects directly (relates in part to Firefox [Bug 1463430](https://bugzilla.mozilla.org/show_bug.cgi?id=1463430)).